src/test/kotlin/org/jetbrains/research/transformations/anonimization/AnonymizationTest.kt

```diff
@@ -0,0 +1,40 @@
+package org.jetbrains.research.transformations.anonimization
+
+import PythonTransformationsTest
+import com.github.gumtreediff.tree.TreeContext
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.io.File
+import java.util.stream.Stream
+
+
+internal class AnonymizationTest : PythonTransformationsTest() {
+
+    companion object {
+        @JvmStatic
+        fun getTestData(): Stream<Arguments> {
+            return Stream.of(
+                *getInAndOutArgumentsArray("data", ::AnonymizationTest)
+            )
+        }
+    }
+
+    @ExperimentalStdlibApi
+    @ParameterizedTest(name = "Apply anonymization test")
+    @MethodSource("getTestData")
+    fun applyAnonymizationTests(inFile: File, outFile: File) {
+        assertCodeTransformation(inFile, outFile, Anonymization::apply)
+    }
+
+    @ExperimentalStdlibApi
+    @ParameterizedTest(name = "Inverse apply anonymization test")
+    @MethodSource("getTestData")
+    fun inverseApplyAnonymizationTests(inFile: File, outFile: File) {
```

Comment:
You don't use outFile, is it ok?

Reply:
Yes, because I should use `inFile -> apply -> reverseApply` and get inFile. OutFile here only for using `getTestData`